test/powershell/Modules/Microsoft.PowerShell.Utility/Import-Alias.Tests.ps1

```diff
@@ -2,82 +2,117 @@
-	    { ipal $pesteraliasfile } | Should -Not -throw
-	}
-
-	It "Should be able to import an alias file and perform imported aliased echo cmd" {
-	    (Import-Alias $pesteraliasfile)
-	    (pesterecho pestertesting) | Should -BeExactly "pestertesting"
-	}
-
-	It "Should be able to use ipal alias to import an alias file and perform cmd" {
-	    (ipal $pesteraliasfile)
-	    (pesterecho pestertesting) | Should -BeExactly "pestertesting"
-	}
+
+    BeforeAll {
+        $newLine = [Environment]::NewLine
+
+        $testAliasDirectory = Join-Path -Path $TestDrive -ChildPath ImportAliasTestDirectory
+        $aliasFilename = "pesteralias.txt"
+        $aliasFilenameMoreThanFourValues = "aliasFileMoreThanFourValues.txt"
+        $aliasFilenameLessThanFourValues = "aliasFileLessThanFourValues.txt"
+
+        $aliasfile = Join-Path -Path $testAliasDirectory -ChildPath $aliasFilename
+        $aliasPathMoreThanFourValues = Join-Path -Path $testAliasDirectory -ChildPath $aliasFileNameMoreThanFourValues
+        $aliasPathLessThanFourValues = Join-Path -Path $testAliasDirectory -ChildPath $aliasFileNameLessThanFourValues
+
+        $commandToAlias = "echo"
+        $alias1 = "pesterecho"
+        $alias2    = '"abc""def"'
+        $alias3    = '"aaa"'
+        $alias4    = '"a,b"'
+
+        # create alias file
+        New-Item -Path $testAliasDirectory -ItemType Directory -Force > $null
+
+        # set header
+        $aliasFileContent = '# Alias File' + $newLine
+        $aliasFileContent += '# Exported by : alex' + $newLine
+        $aliasFileContent += '# Date/Time : Thursday, 12 November 2015 21:55:08' + $newLine
+        $aliasFileContent += '# Computer : archvm'
+
+        # add various aliases
+        $aliasFileContent += $newLine + $alias1 + ',"' + $commandToAlias + '","","None"'
+        $aliasFileContent += $newLine + $alias2 + ',"' + $commandToAlias + '","","None"'
+        $aliasFileContent += $newLine + $alias3 + ',"' + $commandToAlias + '","","None"'
+        $aliasFileContent += $newLine + $alias4 + ',"' + $commandToAlias + '","","None"'
+        $aliasFileContent > $aliasfile
+
+        # create invalid file with more than four values
+        New-Item -Path $testAliasDirectory -ItemType Directory -Force > $null
+        $aliasFileContent = $newLine + '"v_1","v_2","v_3","v_4","v_5"'
+        $aliasFileContent > $aliasPathMoreThanFourValues
+
+        # create invalid file with less than four values
+        New-Item -Path $testAliasDirectory -ItemType Directory -Force > $null
+        $aliasFileContent = $newLine + '"v_1","v_2","v_3"'
+        $aliasFileContent > $aliasPathLessThanFourValues
+    }
+
+    It "Should be able to import an alias file successfully" {
+        { Import-Alias -Path $aliasfile } | Should -Not -Throw
+    }
+
+    It "Should classify an alias as non existent when it is not imported yet" {
+        Get-Alias -Name invalid_alias -ErrorAction SilentlyContinue | Should -BeExactly $null
```

Comment:
This is a negative test, we should actually check for the expected fullyqualified error id.
```suggestion
        {Get-Alias -Name invalid_alias -ErrorAction Stop} | Should -Throw -ErrorId 'ItemNotFoundException,Microsoft.PowerShell.Commands.GetAliasCommand' }
```